cmd/csi-resizer/main.go

```diff
@@ -41,7 +40,7 @@ var (
 	workers      = flag.Int("workers", 10, "Concurrency to process multiple resize requests")
 
 	csiAddress  = flag.String("csi-address", "/run/csi/socket", "Address of the CSI driver socket.")
-	csiTimeout  = flag.Duration("csiTimeout", 15*time.Second, "Timeout for waiting for CSI driver socket.")
+	timeout     = flag.Duration("timeout", 15*time.Second, "Timeout for waiting for CSI driver socket.")
```

Comment:
IMO we should not remove the parameter `csiTimeout`, we should deprecate old parameter and add a new one first

Reply:
Forgot that we havent called out for deprecation yet, will change accordingly.